### PR TITLE
run rustfmt on librustc_typecheck/variance folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: src/librustc_typeck/variance/constraints.rs [hunk 1]
```diff
@@ -144,14 +143,13 @@ impl<'a, 'tcx> ConstraintContext<'a, 'tcx> {
     fn find_binding_for_lifetime(&self, param_id: ast::NodeId) -> ast::NodeId {
         let tcx = self.terms_cx.tcx;
         assert!(is_lifetime(&tcx.map, param_id));
-        match tcx.named_region_map.defs.get(&param_id) {
-            Some(&rl::DefEarlyBoundRegion(_, _, lifetime_decl_id))
-                => lifetime_decl_id,
+        match tcx.named_region_map.get(&param_id) {
```
Comment (srinivasreddy) [inline src/librustc_typeck/variance/constraints.rs]: very interesting. `defs` has been removed from here. cc @nrc

Comment (lqd) [inline src/librustc_typeck/variance/constraints.rs]: Interesting, I ran rustfmt master on the same file, and `defs` was still there (ran it with default settings, so maybe they have some impact here)

Comment (nrc) [inline src/librustc_typeck/variance/constraints.rs]: @srinivasreddy could you also try suing master please? Hopefully this is a bug which has been fixed already

**Comment by alexcrichton** [comment]
Closing due to inactivity, but feel free to resubmit!

**Pull request closed without merging**